java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/models/healthData/RecordExportStatusRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.sagebionetworks.bridge.sdk.models.healthData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.joda.time.DateTime;
+import org.joda.time.LocalDate;
+import org.sagebionetworks.bridge.sdk.Config;
+import org.sagebionetworks.bridge.sdk.models.upload.UploadCompletionClient;
+import org.sagebionetworks.bridge.sdk.models.upload.UploadStatus;
+
+import java.util.List;
+
+/** Represents a record export status request to the Bridge Server. */
+public class RecordExportStatusRequest {
+
+    private List<String> recordIds;
+    private Config.ExporterStatus synapseExporterStatus;
+
+    public List<String> getRecordIds() { return this.recordIds; }
+
+    public Config.ExporterStatus getSynapseExporterStatus() { return this.synapseExporterStatus; }
+
+    @JsonCreator
+    public RecordExportStatusRequest(@JsonProperty("recordIds") List<String> recordIds, @JsonProperty("synapseExporterStatus") Config.ExporterStatus synapseExporterStatus) {
+        this.recordIds = recordIds;
+        this.synapseExporterStatus = synapseExporterStatus;
+    }
+    public void setRecordIds(List<String> recordIds) { this.recordIds = recordIds; }
+
+    public void setSynapseExporterStatus(Config.ExporterStatus synapseExporterStatus) { this.synapseExporterStatus = synapseExporterStatus; }
+}
```

Comment:
We normally add a hashCode and equals method to these classes, and make them final, and verify that they are equal using the EqualsVerifier library. Again I'm not sure how much I care because we're working to remove this SDK in a couple of weeks so probably leave this as is.